src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -325,7 +325,7 @@ export default defineComponent({
                 alignItems: 'center',
                 borderRadius: '2px',
                 border: this.collapseBorder,
-                background: 'var(--dl-color-bg, #30363D)',
+                background: 'var(--dl-color-component, #30363D)',

```

Comment:
Was it reverted on purpose ?
The `component` color is not working in dark mode, I already had to change it.
https://github.com/dataloop-ai/components/commit/dd4cb169eb08af07561e4d6b9513ba4897caa719